Subject: [PATCH 1/1] mtd: rawnand: gigadevice: add support for GD5F1GM7REYIG
===================================================================
--- a/drivers/mtd/nand/raw/nand_ids.c
+++ b/drivers/mtd/nand/raw/nand_ids.c
@@ -92,6 +92,9 @@ struct nand_flash_dev nand_flash_ids[] =
 	{"GD5F1GQ5REYIG SPI NAND 1G",
 		{ .id = {0xc8, 0x41} },
 		SZ_2K, SZ_128, SZ_128K, 0, 2, 128, NAND_ECC_INFO(8, SZ_512), 0},
+	{"GD5F1GM7REYIG SPI NAND 1G 1.8V",
+		{ .id = {0xc8, 0x81} },
+		SZ_2K, SZ_128, SZ_128K, 0, 2, 128, NAND_ECC_INFO(8, SZ_512), 0},
 	{"F50D1G41LB(2M) SPI NAND 1G 1.8V",
 		{ .id = {0xc8, 0x11} },
 		SZ_2K, SZ_128, SZ_128K, 0, 2, 64, NAND_ECC_INFO(4, SZ_512), 0},
